app/src/androidTest/java/com/github/h3lp3rs/h3lp/HelpParametersActivityTest.kt

```diff
@@ -89,6 +92,49 @@ class HelpParametersActivityTest {
         )
     }
 
+    @Test
+    fun clickPhoneButtonAndContactButtonDialsEmergencyContactNumber(){
+        val intent = Intent()
+        val intentResult = ActivityResult(Activity.RESULT_OK, intent)
+        intending(anyIntent()).respondWith(intentResult)
+
+        When(locationManagerMock.getCurrentLocation(anyOrNull())).thenReturn(null)
+        GeneralLocationManager.set(locationManagerMock)
+
+
+        // Clicking on the call for emergency button
+        val phoneButton = onView(withId(R.id.help_params_call_button))
+
+        phoneButton.check(matches(isDisplayed()))
+        phoneButton.perform(click())
+
+        val emergencyContactNumber = "+41216933000"
+
+        val medicalInformation = MedicalInformation(MedicalInformation.MAX_HEIGHT-1,
```

Comment:
Maybe we should add a utility file for tests with common objects, this for example has been created a gazillion times.